ghcide/ghcide.cabal

```diff
@@ -62,7 +62,7 @@ library
     , deepseq
     , dependent-map
     , dependent-sum
-    , Diff                         ^>=0.4.0
+    , Diff                         ^>=0.5
```

Comment:
Seems like this version has nice perf improvements: https://github.com/seereason/Diff/blob/master/CHANGELOG.md

Also needed to bump to avoid adding Diff-0.4 to extra deps because it's not in recent stack resolvers.

Reply:
Small feedback: This change is making trouble in distributing.
There was as I see it no reason to raise the lower bound here. (Raising the upper bound is good.) Especially since Diff 0.4.0 is the current Diff version in stackage-lts this lower bound bump seems a bit hasty.

Reply:
Sorry for that. I didn't think about it too deeply. Do you want me to try to widen the range again?

Reply:
Not sure. Would probably be a good idea, but I have a workaround downstream so I don’t care too much.